pkg/start/start.go

```diff
@@ -296,6 +297,11 @@ func watchHostAgentEvents(ctx context.Context, inst *store.Instance, haStdoutPat
 			} else {
 				logrus.Infof("READY. Run `%s` to open the shell.", LimactlShellCmd(inst.Name))
 			}
+			if inst.StartAtLogin {
+				if err := autostart.CreateFile(runtime.GOOS); err != nil {
```

Comment:
This should happen on creating an instance, not on starting it

Reply:
So if VM wont start would be fine to create autostart file?

Reply:
Yes, should be fine

Reply:
How would I turn on/off autostart for an existing image. We could do this via `limactl edit`, but it will require the user to shut down the instance to enable/disable autostart, which feels clumsy.

Reply:
Ok, I'll move it to create action 👍